storage/src/vespa/storage/storageserver/communicationmanager.cpp

```diff
@@ -230,7 +230,7 @@ CommunicationManager::CommunicationManager(StorageComponentRegister& compReg,
       _mbus(),
       _configUri(configUri),
```

Comment:
If you bring the configUri in the on_configure call, you do not need it in the constructor, nor remember it here.

Reply:
`on_configure` only takes in the actual concrete config object. The `ConfigUri` is used when setting up MessageBus and its merry friends, so looks like it survives for now